concept/type_/plays.rs

```diff
@@ -28,6 +31,8 @@ pub struct Plays<'a> {
 }
 
 impl<'a> Plays<'a> {
+    pub const DEFAULT_CARDINALITY: AnnotationCardinality = AnnotationCardinality::new(1, Some(1));
```

Comment:
Plays has default (0, infinity) cardinality -
`owns` -> `@card(0..1)`
`relates` -> `@card(1..1)`
`plays` -> `@card(0..)`

Reply:
Oh wow, didn't now about it! I'll fix it, but it will take time to fix BDDs in the BDD pr